src/UI/SubtitleWindow.lua

```diff
@@ -58,8 +60,47 @@ function SubtitleWindow.new(): Types.SubtitleWindow
     BackgroundUICorner.Parent = BackgroundFrame
     self.BackgroundUICorner = BackgroundUICorner
 
+    --Connect Nexus VR Core being load.
+    --Nexus VR Character Model is typically loaded from an id, so Nexus VR Core can appear at any time.
+    if UserInputService.VREnabled then
+        if ReplicatedStorage:FindFirstChild("NexusVRCore") then
+            self:LoadNexusVRCore()
+        else
+            ReplicatedStorage.ChildAdded:Connect(function(Child)
+                if Child.Name ~= "NexusVRCore" then return end
+                self:LoadNexusVRCore()
+            end)
+        end
+    end
+
     --Return the object.
-    return self
+    return self :: any
```

Comment:
`self` should be typed as `{}`, not as `any`. Similarly, to the previous comments, a custom type can solve this problem.